### PR TITLE
Automatically map built-in touchscreens/tablets to built-in panels 

### PR DESCRIPTION
**Pull request by emersion**
Detect whether an output is built-in via its type. Detect whether
a touchscreen/tablet is built-in via its ID_PATH property.

### CODE REVIEW

**Comment by emersion** [comment]
For reference, the mutter logic for this is a lot more involved: https://gitlab.gnome.org/GNOME/mutter/-/blob/662e29990a1deb1e70125f5f00a184e2ce825917/src/backends/meta-input-mapper.c#L471

Only matching built-in devices allows us to have less false-positives, I hope.

**Comment by emersion** [comment]
Tested with a laptop which has an integrated touchscreen, and with an external Wacom tablet via USB.

**Review by Xyene — APPROVED** [review]
Code LGTM, thanks! 

> Only matching built-in devices allows us to have less false-positives, I hope.

This seems like a reasonable trade-off for now. If mutter's approach pans out long-term, we can always revisit this.

> Tested with a laptop which has an integrated touchscreen, and with an external Wacom tablet via USB.

I don't have any hardware to test on right now, but the change seems straightfoward enough.

File: sway/input/seat.c [hunk 1]
```diff
@@ -666,6 +667,36 @@ static void seat_reset_input_config(struct sway_seat *seat,
...
+}
+
+static const char *get_builtin_output_name(void) {
+	const char *match = NULL;
+	for (int i = 0; i < root->outputs->length; ++i) {
+		struct sway_output *output = root->outputs->items[i];
+		const char *name = output->wlr_output->name;
+		if (has_prefix(name, "eDP-") || has_prefix(name, "LVDS-") ||
+				has_prefix(name, "DSI-")) {
+			if (match != NULL) {
```
[the remark this answers is not in the document]
Comment (emersion) [reply sway/input/seat.c:685]: Good idea, added some comments!

**Comment by Xyene** [comment]
Thanks!

**Pull request merged**